src/setup.js

```diff
@@ -264,6 +324,10 @@ function convertTypeToGQL(implicitCollection, propertyValue){
     return propertyValue
 }
 
+function generateSchemaStr(defs){
```

Comment:
Dead code? I think my PR removed this helper

Reply:
Yes, good catch.